src/store/wallet/effects/send/send.ts

```diff
@@ -360,7 +360,7 @@ export const getInvoiceEffectiveRate =
     const invoiceCurrency = getCurrencyCodeFromCoinAndChain(coin, chain);
     return (
       precision &&
-      invoice.price /
+      invoice.usdAmount /
```

Comment:
this change will break the effective rate calculation for other fiat currencies like EUR, GBP, etc and is not needed